kyuubi-server/src/main/scala/org/apache/kyuubi/engine/spark/SparkBatchProcessBuilder.scala

```diff
@@ -50,8 +50,16 @@ class SparkBatchProcessBuilder(
     KyuubiApplicationManager.tagApplication(batchId, "spark", clusterManager(), batchKyuubiConf)
 
     (batchKyuubiConf.getAll ++ sparkAppNameConf()).foreach { case (k, v) =>
+      val newKey = {
+        // convert the kyuubi config with `spark` prefix, so that spark could identify
+        if (k.startsWith("kyuubi.")) {
+          "spark." + k
```

Comment:
shall we reuse the code in `SparkProcessBuilder` ? so we can keep the same behavior on spark conf and hadoop conf.

Reply:
sorry for the late reply, updated.